### PR TITLE
Create lib errors popover

### PR DESCRIPTION
**Pull request by yipingliao**
@andrewconner Create library and manage library now has error popovers on the name field. Heavily "borrowed" styles from your registration modal. :)
https://app.asana.com/0/15523651812135/18373253983183

Also fixed a tiny styling bug: 
https://app.asana.com/0/15523651812135/18383371984383

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.styl [hunk 1]
```diff
@@ -273,6 +272,7 @@
 .manage-lib-member
   position: relative
   box-sizing: border-box
+  margin: 0 auto
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.styl:275]: @ahsu1230 This centers that `member` li element (this is in lieu of `position: relative` and `left: 30px`)

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.tpl.html [hunk 2]
```diff
@@ -25,16 +25,16 @@ <h1 class="dialog-header">
...
             </div>
             <input class="manage-lib-input manage-lib-name-input ng-pristine ng-valid" ng-model="library.name" type="text" name="name" placeholder="Type a name for your library..." autocomplete="off" novalidate>
+            <span ng-show="$error.name" class="manage-lib-name-error">{{$error.name}}</span>
             <span class="manage-lib-slug">kifi.com/{{username}}/
               <input ng-change="editSlug()" class="manage-lib-slug-input ng-pristine ng-valid" ng-model="library.slug" ng-class="{ 'slug-edited': userHasEditedSlug }" type="text" name="slug" placeholder="new-library" novalidate>
             </span>
           </div>
 
           <div class="manage-lib-input-wrapper">
-            <span class="manage-lib-heading manage-lib-input-title manage-lib-description-title-input">Description<span class="manage-lib-error" ng-if="$error.description">{{$error.description}}</span></span><!-- for removing space between inline-block --><textarea
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.tpl.html:37]: We actually don't have any checks for description errors, so this is extraneous.

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.styl (hunk 1 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.styl:275]: I see :+1:

**Comment by andrewconner** [comment]
Nice, thanks

**Comment by ahsu1230** [comment]
looks good!

**Comment by yipingliao** [comment]
Closing to fix merge conficts.

**Pull request closed without merging**